### DISCUSSION TITLE
Android Manifest error

### DISCUSSION THREAD

**Discussion by AmirSinaRZ** [Sketchware-Pro/Sketchware-Pro#766] [Q&A] [answered]
When I add this code to android manifest sketchware says unbound prefix.

**Comment by AmirSinaRZ** [comment]
<provider
			android:name="androidx.startup.InitializationProvider"
			android:authorities="${applicationId}.androidx-startup"
			android:exported="false"
			tools:node="merge">
			<meta-data
				android:name="io.github.solrudev.simpleinstaller.SimpleInstallerInitializer"
				android:value="androidx.startup" />
		</provider>

		<receiver
			android:name=".impl.installer.InstallationEventsReceiver"
			android:exported="false" />

**Comment by AmirSinaRZ** [comment]
provider tag is problem

**Comment by j-px** [comment, marked as answer]
Remove `tools:node="merge"`.

**Reply by AmirSinaRZ** [reply]
Thanks
